releasenotes/notes/logs-fix-empty-log-with-docker-container-64129b051c554ad4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Empty logs could be appear when collecting Docker logs in addition
```

Comment:
I would give that much technical details and just say `Fix an issue where empty logs could be sent when collection Docker logs`